content/join.hbs

```diff
@@ -59,32 +59,32 @@
 			<div class="columns">
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">November</div>
+						<div class="month">July</div>
 						<div class="day">15</div>
 					</div>
 					<p>Cross-registration Deadline</p>
 				</div>
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">January</div>
-						<div class="day">5</div>
+						<div class="month">July</div>
+						<div class="day">19</div>
 					</div>
 					<p>NSE Package Due</p>
 				</div>
 				<div class="column date">
 					<div class="calendar">
-						<div class="month">January</div>
-						<div class="day">14</div>
+						<div class="month">July</div>
+						<div class="day">5</div>

```

Comment:
Just want to confirm that In-Processing is July 5th which is before either of the other enrollment deadlines?

Reply:
That was a confirmed error, I thought I fixed it to Aug 5, thank you!